### PR TITLE
update error message, delete a redundant word

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/eviction/eviction_manager.go [hunk 1]
```diff
@@ -240,7 +240,7 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 	updateStats := true
 	summary, err := m.summaryProvider.Get(updateStats)
 	if err != nil {
-		glog.Errorf("eviction manager: failed to get get summary stats: %v", err)
+		glog.Errorf("eviction manager: failed to get summary stats: %v", err)
```
Comment (nikhita) [inline pkg/kubelet/eviction/eviction_manager.go]: @zhangxiaoyu-zidif thanks for catching this! :)

**Comment by nikhita** [comment]
/assign @derekwaynecarr 
for approval

**Review by feiskyer — COMMENTED** [review]
/lgtm

**Comment by nikhita** [comment]
@Random-Liu can you approve? This is a simple one to approve! :)

**Comment by nikhita** [comment]
/retest

so many retests. :(

**Comment by nikhita** [comment]
/assign @vishh 
for approval

**Pull request closed without merging**